### ISSUE TITLE
Spike: Investigate how env variables are set to cloud environments.

### ISSUE THREAD

**Issue by n0remac** [raft-tech/TANF-app#2261]
**Description:**

**Background:**

When code is deployed to a cloud gov environment variables are set on the cloud gov instances. These variables originate from the Circle Ci environment variables, a .env file, or are constructed by the cicd pipeline. The deploy frontend and deploy backend scripts contain much of the logic for how the variables are transferred to the cloud gov instances. There are instances where the app can break due to old variables not being removed.

**Action:**
As a developer I want the deployment code to be easy to understand. The env variables used by the pipeline need to be reliably set in the cloud gov environment. This spike is meant to investigate how we currently set environment variables in cloud gov with our deployment pipeline and determine a cleaner path forward.

**AC:**
- [ ] Understand current deployment code 
- [ ] Ask what could go wrong
- [ ] Ask how state could be set more reliably
- [ ] Determine how code can be made simpler
- [ ] Determine how code can be made more reliable

**Tasks:**
- [ ] Document findings in issue or hack md
- [ ] Create plan of attack and get team buy in
- [ ] Create a new Issue

**Comment by andrew-jameson** [comment]
Won't do, will reorganize after #2260 and create a new ticket with more definitive goals
